proto/github.proto

```diff
@@ -8,6 +8,9 @@ package github;
 // associated with the group selected in the request context.
 message UnlinkGitHubAccountRequest {
   context.RequestContext request_context = 1;
+
+  // Whether to unlink the user-level account.
+  bool user = 2;
```

Comment:
nit: maybe unlink_user_account is a clearer name for this?

Reply:
Done